src/applications/static-pages/createMyVALoginWidget.js

```diff
@@ -0,0 +1,19 @@
+import { isLoggedIn } from '../../platform/user/selectors';
+
+export default function createMyVALoginWidget(store) {
+  const root = document.getElementById('myva-login');
+  const homePageStoreListener = () => {
+    if (root && isLoggedIn(store.getState())) {
```

Comment:
Decided to forgo react for this since it's just a static content swap. There are other examples similar to this in other places.